tests/e2e/scenarios/02_upgrade.js

```diff
@@ -38,7 +38,8 @@ Upgrade
 Log out
 Check new version
  */
-describe(`Upgrade PrestaShop from '${global.PS_VERSION}' to '${global.PS_VERSION_UPGRADE_TO}'`, async () => {
+// eslint-disable-next-line max-len
+describe(`[${global.AUTOUPGRADE_VERSION}] Upgrade PrestaShop from '${global.PS_VERSION}' to '${global.PS_VERSION_UPGRADE_TO}'`, async () => {
```

Comment:
This is a hack to make tests look like the capture since the nightly site can't handle how ti insert and display 2 module's branches.
![image](https://user-images.githubusercontent.com/48441421/132857759-543a5812-d29f-4ff2-9f54-f1e93457cd59.png)